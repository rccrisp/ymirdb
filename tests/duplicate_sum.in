set a 1
set b a a 3
sum b
pop b
sum b
bye