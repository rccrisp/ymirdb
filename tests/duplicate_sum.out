> ok

> ok

> 5

> a

> 4

> bye
